style/colors.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --primary-font-color: #0f141a;
+  --secondary-font-color: #546373;
+
+  --placeholder-font-color: #989797;
+
+  --search-bar-focus-b-color: #3d91f9;
```

Comment:
Nice use of variables

Reply:
Thanks Nathan for your feedback and your instructions.
After we have studied semantic elements today I'll replace all unnecessary divs with the semantics one.